index.html

```diff
@@ -251,14 +251,15 @@ <h6>Skills</h6>
             <form id="form" action="https://formspree.io/f/mwkdkopo" method="POST">
                 <fieldset class="contactContainer">
                     <input class="textinput" name="name" type="text" id="name" placeholder="Full Name"
-                        maxlength="30" required />
+                        maxlength="30" pattern="[a-z]" required />
```

Comment:
- [ ] Great job implementing the validation :clap: However the form is not sending when entering a valid full name. Kindly check the `pattern` or simply remove it.
![Screenshot from 2023-07-25 09-09-38](https://github.com/DevKaranJ/DevPortfolio/assets/91142432/8ffb6cd7-f511-4011-9e27-7e2c90c7b27c)